### PR TITLE
Connect to MongoDB

### PR DESCRIPTION
**Pull request by z-cohn**

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true,
+        lowercase: true
+    },
+
+    email: {
```
Comment (z-cohn) [inline models/user.js:10]: add unique: true to email and username

**Review by Jacob-Levitan — COMMENTED** [review]
See inline comment

File: models/user.js (hunk 1 above)
Comment (z-cohn) [reply models/user.js:11]: I was doing it for simplicity with the database. But sure we can change it to normal. Btw this setting doesn't "allow/disallow" anything, just converts it to lowercase before saving

**Review by Jacob-Levitan — COMMENTED** [review]
lgtm

**Pull request merged**